File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/CboAvgTimeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
	public class CboAvgTimeInfo
	{
		private String _timeTerm;
		private String _timeTerm2;

		public String TimeTerm
		{
			get { return this._timeTerm; }
			set { this._timeTerm = value; }
		}

		public String TimeTerm2
		{
			get { return this._timeTerm2; }
			set { this._timeTerm2 = value; }
		}

		public CboAvgTimeInfo() { }

		public CboAvgTimeInfo(String timeTerm, String timeTerm2)
		{
			this._timeTerm = timeTerm;
			this._timeTerm2 = timeTerm2;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/CboAvgTimeInfo.java
package nta.med.data.model.ihis.nuro;

public class CboAvgTimeInfo {
	private String timeTerm;
	private String timeTerm2;

	public CboAvgTimeInfo(String timeTerm, String timeTerm2) {
		super();
		this.timeTerm = timeTerm;
		this.timeTerm2 = timeTerm2;
	}

	public String getTimeTerm() {
		return timeTerm;
	}

	public void setTimeTerm(String timeTerm) {
		this.timeTerm = timeTerm;
	}

	public String getTimeTerm2() {
		return timeTerm2;
	}

	public void setTimeTerm2(String timeTerm2) {
		this.timeTerm2 = timeTerm2;
	}

}